nexus_constructor/line_geometry.py

```diff
@@ -0,0 +1,27 @@
+from PySide2 import QtCore
+from PySide2.Qt3DRender import Qt3DRender
+
+
+class LineGeometry(Qt3DRender.QGeometry):
+    def __init__(self, vertex_data: QtCore.QByteArray, parent=None):
+        """
+        Geometry for depicting a line. Used for axes placed in the origin of the instrument view.
+        :param vertex_data: A byte array containing the coordinates for the start and end points of the line.
+        :param parent: The object parent.
+        """
+        Qt3DRender.QGeometry.__init__(self, parent)
+        self.m_position_buffer = Qt3DRender.QBuffer(self)
```

Comment:
Can you add some comments to describe what's going on here? Also, what does the `m_` prefix signify?